.gitignore

```diff
@@ -1,3 +1,4 @@
 /target
 **/*.rs.bk
 Cargo.lock
+.vscode/
```

Comment:
This is wrong, we cannot add ignores for all ides out there. You should use global ignores for this.